src/shared/renderUtils.jsx

```diff
@@ -1,19 +1,60 @@
 import React from "react";
 
-function renderBody(body){
+function renderBody(body, scope = "", className = ""){
 	const paragraphs = body.split("\n"); // splits the string at every line-break
-	return paragraphs.map(renderParagraph);
+
+	return paragraphs.map(
+		renderParagraph.bind(this, scope, className)
+	);
 }// renderBody
 
-function renderParagraph(text, index){
-	const key = `about__body__paragraph_${index}`;
+function renderParagraph(scope, className, text, index){
```

Comment:
as with my comments on `renderBody`, let's create an `options` object for this, otherwise it's gonna get out of hand.